src/Components/CreateImageWizard/steps/Packages/PackageRecommendations.tsx

```diff
@@ -213,7 +215,12 @@ const PackageRecommendations = () => {
                           <Button
                             variant="link"
                             component="a"
-                            onClick={() => addRecommendedPackage(pkg)}
+                            onClick={() => {
+                              analytics.track('recommended-package-added', {
+                                pkgName: pkg,
```

Comment:
```suggestion
                                pkgName: pkg,
                                selectedPackages: packages.map(pkg => pkg.name),
                                shownRecommendations: data.packages
```
@regexowl not sure if what I suggested works as expected, but I'd like to also track the packages that the user has already selected prior to adding a recommended package as well as the recommendations that are displayed to them prior to adding the recommended package.

Reply:
Works great, thank you! The PR is updated, the full list of previously selected packages and the list of recommendations should be tracked as well now :)